variants/aws-dev/Cargo.lock

```diff
@@ -1,5 +1,504 @@
 # This file is automatically @generated by Cargo.
 # It is not intended for manual editing.
+[[package]]
+name = "acpid"
+version = "0.1.0"
```

Comment:
We can delete these individual Cargo.lock files from the variant directories, since they should be superseded by the workspace lock.